### DISCUSSION TITLE
New graphics settings

### DISCUSSION THREAD

**Discussion by CoolGuy12490** [ppy/osu#18929] [Ideas] [answered]
1. Determines if the animations in the circles with the new skin are on or off
2. Determines if glow is on or off (you can choose where it is)
3. Three presets for effects: performance, medium, and perfect

**Comment by bdach** [comment, marked as answer]
if this proposal is related to low performance, then please give us a chance to optimise what we have now first rather than add options to degrade the visual quality of the game.

**Reply by CoolGuy12490** [reply]
Ok
